Source/CesiumRuntime/Private/CesiumGeoreferenceComponent.cpp

```diff
@@ -83,6 +87,13 @@ void UCesiumGeoreferenceComponent::SnapToEastSouthUp() {
         *this->GetName());
     return;
   }
+
+  // If our ECEF position isn't valid, we need to compute it from the Actor's
+  // Transform first.
+  if (!this->_ecefIsValid) {
```

Comment:
Can this function ever be called when the flag is `false`?

Reply:
Probably not very likely, but any C++ code with a pointer to the component could call it at any inopportune time it wants. I'm not sure if similar could be done from Blueprints.